source/MRMesh/MRMeshFwd.h

```diff
@@ -295,6 +295,10 @@ MR_CANONICAL_TYPEDEFS( (template <typename T> struct), Plane3,
 )
 
 MR_CANONICAL_TYPEDEFS( (template <typename V> struct MRMESH_CLASS), Box,
+    ( Box1i,  Box<int>       )
+    ( Box1ll, Box<long long> )
+    ( Box1f,  Box<float>     )
+    ( Box1d,  Box<double>    )
```

Comment:
Mb also add MinMax alias?

Reply:
`MinMax.min` and `MinMax.max` look strange